test/FunctionalTests/Microsoft.OData.Edm.Tests/ExtensionMethods/TargetHelperTests.cs

```diff
@@ -0,0 +1,216 @@
+﻿
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.OData.Edm.Tests.ExtensionMethods
+{
+    public class TargetHelperTests
+    {
+        [Fact]
+        public void GetTargetSegmentsForEntitySetProperty()
+        {
+            IEdmModel edmModel = new TargetPathSampleModel().Model;

```

Comment:
>IEdmModel edmModel = new TargetPathSampleModel().Model; [](http://example.com/codeflow?start=12&length=55)

If there's no test cases changing the model, it's better to use a static field to cache the model, so we don't need to generate the model for all test cases.